index.html

```diff
@@ -30,8 +31,9 @@
 
     <h1 id="mystery"></h1>
 
-    <button>REVEAL THE CRIME</button>
+    <button type="button" id="revealMystery" onclick="revealMystery()">REVEAL THE CRIME</button>
```

Comment:
Here you've added the `onclick="revealMystery()"` which is great. This means you don't have to add this in the js file if you've already written it here. I'm not sure what the `type="button"` does or if you need it as you've already defined a button element? Maybe the teacher can confirm?

Reply:
That's a lot of duplicating logic. You need to decide which approach you want to use Destiny